astropy/units/core.py

```diff
@@ -782,11 +787,11 @@ def is_equivalent(self, other, equivalencies=[]):
 
         Parameters
         ----------
-        other : unit object or string or tuple
+        other : :class:`~astropy.units.Unit` or string or tuple
```

Comment:
The use of `:class:` is a bit inconsistent; I think in this context only classes are expected, so it is superfluous. (and makes the in-line help lines longer without benefit).

Reply:
✅